src/main/java/spoon/refactoring/CtRenameLocalVariableRefactoring.java

```diff
@@ -85,8 +85,7 @@ public void accept(CtReference t) {
 		@Override
 		public ScanningMode enter(CtElement element) {
 			if (ignoredParent != null && element instanceof CtElement) {
-				CtElement ele = (CtElement) element;
-				if (ele.hasParent(ignoredParent)) {
+				if (((CtElement) element).hasParent(ignoredParent)) {
```

Comment:
I really don't agree with this change: generally I'm the one extracting those casts in a local variable. I find it really more readable, and it's really useful when debugging. 

Now here I even don't understand why a cast is needed: element is already a CtElement, so we just need to remove the cast, don't we?

Reply:
@surli 
> I really don't agree with this change: generally I'm the one extracting those casts in a local variable. I find it really more readable, and it's really useful when debugging.

Restored.

> Now here I even don't understand why a cast is needed: element is already a CtElement, so we just need to remove the cast, don't we?

We do :-)